resources/definitions/ultimaker_s3.def.json

```
@@ -9,7 +9,11 @@
         "manufacturer": "Ultimaker B.V.",
         "file_formats": "application/x-ufp;text/x-gcode",
         "platform": "ultimaker_s3_platform.obj",
-        "bom_numbers": [213482, 213483],
+        "bom_numbers": [
+            213482,
+            213483,
+            213484
```

```suggestion
            213484,
```

Less of a diff in the future if bom numbers are added